pkg/utils/io.go

```diff
@@ -28,3 +33,11 @@ func (rs *ReaderSource) Read(p []byte) (n int, err error) {
 	rs.buf = rs.buf[n:]
 	return n, rs.err
 }
+
+// Close something in a defer for errcheck
+func Close(c io.Closer) {
+	err := c.Close()
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Oh, no! `log.Fatal` will exit the program: https://golang.org/pkg/log/#Logger.Fatal

Reply:
Perhaps, you wanted to use something like `zap.S().Error`.